addons/udes_stock/models/stock_picking.py

```diff
@@ -601,6 +601,11 @@ def update_picking(
             # validate stock.picking
             picking.action_done()  # old do_transfer
 
+        self.env.ref('udes_stock.picking_update').with_context(
```

Comment:
Big method with 14 optional args... This is not the only path of execution :(
Are we sure we don't want to consider also the case where the method gets `quant_ids` (creates "extra stock.moves to the picking" and return `True`)?
On the other hand, for the above `if` clause where an error is thrown, I think it's ok not to trigger the update.

Reply:
Due to the nature of the update_picking method I agree it's okay not to trigger the update in this case; as you say it's a separate execution path.